docs/content/guides/recipes/howto-dev-recipes/index.md

```diff
@@ -55,15 +55,21 @@ Local development environments created by the rad init command include a set of
 
    When a Recipe is named "default" it will be used automatically when a resource doesn't specify a Recipe name. This makes it easy for applications to fully defer to the Environment for how to manage infrastructure.  
 
-## Step 2: Define your application
+## Step 2: Create a `bicepconfig.json` in your application's directory 
```

Comment:
Also updated this so it's consistent. This already has a select "no" for app setup @Reshrahim